internal/distro/rhel84/distro.go

```diff
@@ -0,0 +1,1107 @@
+			"iwl7260-firmware",
+		},
+		excludedPackages: []string{
+			"subscription-manager",
+		},
+		enabledServices: []string{
+			"NetworkManager.service", "firewalld.service", "rngd.service", "sshd.service",
+			"greenboot-grub2-set-counter", "greenboot-grub2-set-success", "greenboot-healthcheck",
+			"greenboot-rpm-ostree-grub2-check-fallback", "greenboot-status", "greenboot-task-runner",
+			"redboot-auto-reboot", "redboot-task-runner",
+		},
+		rpmOstree: true,
+		assembler: func(uefi bool, options distro.ImageOptions, arch distro.Arch) *osbuild.Assembler {
+			return ostreeCommitAssembler(options, arch)
+		},
+	}
+	amiImgType := imageType{
+		name:     "ami",
+		filename: "image.raw",
+		mimeType: "application/octet-stream",
+		packages: []string{
+			"checkpolicy",
+			"chrony",
+			"cloud-init",
+			"cloud-init",
+			"cloud-utils-growpart",
+			"@core",
+			"dhcp-client",
+			"gdisk",
+			"insights-client",
+			"kernel",
+			"langpacks-en",
+			"net-tools",
+			"NetworkManager",
+			"redhat-release",
+			"redhat-release-eula",
+			"rng-tools",
+			"rsync",
+			"selinux-policy-targeted",
+			"tar",
+			"yum-utils",
+
+			// TODO this doesn't exist in BaseOS or AppStream
+			// "rh-amazon-rhui-client",
+		},
+		excludedPackages: []string{
+			"aic94xx-firmware",
+			"alsa-firmware",
+			"alsa-lib",
+			"alsa-tools-firmware",
+			"biosdevname",
+			"dracut-config-rescue",
+			"firewalld",
+			"iprutils",
+			"ivtv-firmware",
+			"iwl1000-firmware",
+			"iwl100-firmware",
+			"iwl105-firmware",
+			"iwl135-firmware",
+			"iwl2000-firmware",
+			"iwl2030-firmware",
+			"iwl3160-firmware",
+			"iwl3945-firmware",
+			"iwl4965-firmware",
+			"iwl5000-firmware",
+			"iwl5150-firmware",
+			"iwl6000-firmware",
+			"iwl6000g2a-firmware",
+			"iwl6000g2b-firmware",
+			"iwl6050-firmware",
+			"iwl7260-firmware",
+			"libertas-sd8686-firmware",
+			"libertas-sd8787-firmware",
+			"libertas-usb8388-firmware",
+			"plymouth",
+
+			// TODO this cannot be removed, because the kernel (?)
+			// depends on it. The ec2 kickstart force-removes it.
+			// "linux-firmware",
+
+			// TODO setfiles failes because of usr/sbin/timedatex. Exlude until
+			// https://errata.devel.redhat.com/advisory/47339 lands
+			"timedatex",
+		},
+		defaultTarget: "multi-user.target",
+		kernelOptions: "ro console=ttyS0,115200n8 console=tty0 net.ifnames=0 rd.blacklist=nouveau nvme_core.io_timeout=4294967295 crashkernel=auto",
+		bootable:      true,
+		defaultSize:   6 * GigaByte,
+		assembler: func(uefi bool, options distro.ImageOptions, arch distro.Arch) *osbuild.Assembler {
+			return qemuAssembler("raw", "image.raw", uefi, options, arch)
+		},
+	}
+
+	qcow2ImageType := imageType{
+		name:     "qcow2",
+		filename: "disk.qcow2",
+		mimeType: "application/x-qemu-disk",
+		packages: []string{
+			"@core",
+			"chrony",
+			"dnf",
+			"dosfstools",
+			"efi-filesystem",
+			"efivar",
+			"efivar-libs",
+			"grub2-efi-x64",
+			"shim-x64",
```

Comment:
@gicmo I added these files to the qcow2 image type to follow what is supported in the official image. All x86_64 images will have grub2-efi-x64 and shim-x64 in their build packages. Do you have thoughts on which image types might need these packages installed? The official kickstarts do not explicitly install these packages.